FireboltNETSDK/FireboltClient.cs

```diff
@@ -43,11 +43,11 @@ public class FireboltClient
     private readonly string _jsonContentType = "application/json";
     private readonly string _textContentType = "text/plain";
 
-    public FireboltClient(String id, String secret, String endpoint, String? env, HttpMessageInvoker httpClient)
+    public FireboltClient(String id, String secret, String? endpoint, String? env, HttpMessageInvoker httpClient)
     {
         _httpClient = httpClient;
         _settings = new Lazy<JsonSerializerSettings>(new JsonSerializerSettings());
-        _endpoint = env != null ? $"api.{env}.firebolt.io" : endpoint;
+        _endpoint = env != null ? $"api.{env}.firebolt.io" : endpoint!;
```

Comment:
IMO, we should do a check here that either `env` or `endpoint` parameters passed in are not null and raise an exception if it isn't the case. Otherwise with the null forgiving operator we're falsely declaring `_endpoint` to never contain null only to fail somewhere down the line.

Reply:
As far as I understand endpoint is not needed at all. I planned to remove it in separate PR. What do you think?

Reply:
Let's discuss if it's needed on a daily tomorrow. I'd prefer to keep the parameters consistent (as much as we can) across the drivers.
For this PR, let's aim to keep the changes self-contained. If your priorities shift and we never get around to removing it at least let's keep this in a good state. I think an error check for both `env` and `endpoint` not being null will not hurt us or add too much development overhead.

Reply:
Sire. This was the reason that I have not changed anything here except adding nullability to the endpoint. I can remove it (hopefully). I really do not remember why I added it. In this case nothing will be changed.

Reply:
That also works, we probably don't want pointless changes bundled in.

Reply:
Well, I have removed the nullability. It is really not needed.